### PR TITLE
Finalize project

### PR DESCRIPTION
**Pull request by BenjaminGarza**

### CODE REVIEW

File: spec/main-spec.rb [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      end
+
+      it "checks if player has won by diagonals" do
+        @board.current_state = [["X","1,2","1,3"],["2,1","X","2,3"],["3,1","3,2","X"]]
+        expect(@board.win_check("X")).to eql(true)
+
+        @board.current_state = [["1,1","1,2","X"],["2,1","X","2,3"],["X","3,2","3.3"]]
+        expect(@board.win_check("X")).to eql(true)
+      end
+    end
```
Comment (maelfosso) [inline spec/main-spec.rb]: About testing `Board` class
- Test the initialize method
- Add negative scenario

Comment (maelfosso) [reply spec/main-spec.rb]: What about the **Draw** state ?

**Review by maelfosso — CHANGES_REQUESTED** [review]
Hello,

You really master how to write test. The result is so awesome !!! 

Before moving forward there are some things that should be improved.
- Please it's not `main-spec.rb` but `main_spec.rb`
- Generally, the tests are separated by class. one file for one class
- Your tests should be improved with **negative scenario**
Also, read the comments below.

Kindly make the suggested changes and re-submit a code review request.
Thanks.

**Review by Ispirett — APPROVED** [review]
Great job with changes made and your porject @BenjaminGarza  @mapra99 :1st_place_medal: :100: 
you can merge now!

#### Optional not required
Its good practice to use `let(varible_name) {Border.new } for initiailzations`
Also, this helps make you test output more readable `--format documentation` 
you can add it to your .rspec file

#### Git/Githubflow reminder.

Friendly reminder: You should use Github/Git Flow in this project. However, changing it now would take too much time.
Please make sure you use Github/Git Flow in the next project. You will find instructions in the Patwright lesson description.

**Pull request merged**